doc/sphinxman/source/methods.rst

```diff
@@ -53,6 +53,7 @@ A complete list of standard keywords is provided in Appendix
    mp2f12
    cc
    fnocc
+   dlpnocc
```

Comment:
also edit introduction.rst . there's a dlpno column in the table at the bottom. and if you want to add a dlpno section with your paper in the references part further up.

Reply:
Done!